### ISSUE TITLE
Question about the 302_classification.py

### ISSUE THREAD

**Issue by happsky** [MorvanZhou/PyTorch-Tutorial#16]
Thanks for sharing this tutorial code. 
One question is that your 302_classification.py file does not contain a 'F.softmax()' function, which is inconsistent with the situation in your Youtube video.﻿

**Comment by MorvanZhou** [comment]
Pytorch does the softmax itself. Here you can find the explaination http://pytorch.org/docs/master/nn.html#crossentropyloss
